website/docs/r/cdn_frontdoor_rule.html.markdown

```diff
@@ -143,8 +143,6 @@ resource "azurerm_cdn_frontdoor_rule" "example" {
 
 ## Arguments Reference
 
-The following arguments are supported:
```

Comment:
why's this been removed/can we add this back in?

Reply:
Fixed.